service/src/main/java/de/starwit/mapper/AppMapper.java

```diff
@@ -11,69 +11,68 @@
 import de.starwit.dto.EntityDto;
 import de.starwit.persistence.entity.App;
 import de.starwit.persistence.entity.AppTemplate;
+import de.starwit.persistence.repository.AppTemplateRepository;
 
 @Component
 public class AppMapper implements Serializable, CustomMapper<App, AppDto> {
 
-  @Autowired
-  EntityMapper entityMapper;
-  
-  private static final long serialVersionUID = 1L;
-  public static final long defaultAppTemplateID = 1L;
+    @Autowired
+    EntityMapper entityMapper;
 
-  public AppDto convertToDto(App entity) {
-    AppDto dto = null;
-    if (entity != null) {
-      dto = new AppDto();
-      dto.setId(entity.getId());
-      dto.setBaseName(entity.getTitle());
-      dto.setPackageName(entity.getPackagePrefix());
-      dto.setEntities(entityMapper.convertToDtoList(entity.getDomains()));
-      dto.setGroupsToAssign(entity.getGroups());
-      AppTemplateDto appTemplateDto = new AppTemplateDto();
-      if (entity.getTemplate() != null) {
-        appTemplateDto.setId(entity.getTemplate().getId());
-        appTemplateDto.setName(entity.getTemplate().getTemplateName());
-        appTemplateDto.setCredentialsRequired(entity.getTemplate().isCredentialsRequired());
-        dto.setTemplate(appTemplateDto);
-      }
+    private static final long serialVersionUID = 1L;
+    public static final long defaultAppTemplateID = 1L;
```

Comment:
Es wäre glaube ich noch wichtig, eine Löschung dieses Templates zu verhindern. Ansonsten läuft man in das gleiche Problem wie vorher...